kie-dmn-bom/pom.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@tarilabs 
Same comment here for you :)

Reply:
a move from -build-bootstrap to -knowledge (ad interim).
Good for me thanks Daniele.